### PR TITLE
docs(sass): add guide for carbon-components sass

### PR DESCRIPTION
**Pull request by joshblack**
Adds a quick guide on the sass capabilities of `carbon-components`.

#### Changelog

**New**

- `docs/guides/sass.md`

**Changed**

**Removed**

### CODE REVIEW

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @joshblack!

**Review by emyarod — APPROVED** [review]
is the FAQ meant to empty at the moment? in any case, looks good to me

File: docs/guides/sass.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+| ----------------- | -------------------------------------------------------------------- | ------------- |
+| `$css--font-face` | Includes the font-face mixins for the current font family (IBM Plex) | `true`        |
+| `$css--helpers`   | Includes classes and utilities that are commonly used by components  | `true`        |
+| `$css--body`      | Sets a top-level reset, type style, and color for the `<body>` tag   | `true`        |
+| `$css--use-layer` | Enables use of box-shadow in `layer()` helpers                       | `true`        |
+| `$css--reset`     | Includes a top-level CSS Reset                                       | `true`        |
+
+## Feature flags
+
+The Carbon Design System takes advantage of feature flags to conditionally
```
Comment (elizabethsjudd) [inline docs/guides/sass.md:80]: Is there a list of what all of the available feature flags are at a given time?

Comment (joshblack) [reply docs/guides/sass.md:80]: Good question! It should be showing up in our generated sassdoc but I'm not seeing it, let me bug @mattrosno to figure out why 😂

Comment (mattrosno) [reply docs/guides/sass.md:80]: On it!

**Pull request merged**